commercial/app/controllers/Multi.scala

```diff
@@ -128,7 +128,7 @@ class Multi(bestsellersAgent: BestsellersAgent,
         Cached(componentMaxAge) {
           JsonComponent(JsArray((offerTypes zip contents).map { case (contentType, content) => Json.obj(
               "type" -> contentType,
-              "value" -> Json.toJson(content)(Merchandise.writes)
+              "value" -> Json.toJson(content)(Merchandise.merchandiseWrites)
```

Comment:
Do you need to specify the writes implementation?  It should be able to work it out.